.gitignore

```diff
@@ -11,6 +11,7 @@
 
 # Virtualenv
 /env
+.direnv/
```

Comment:
What is `.direnv/`? Does this belong in this commit?

Reply:
It's a virtual env that auto activates when you `cd` at the root of the repo or below: https://direnv.net/
Not sure if it belongs to this PR, it's just something I use, but I can remove it if you prefer.

Reply:
Yeah, it probably doesn't belong in this commit. You could keep this in your global `.gitignore` file in `~/.config/git/ignore`. I do this with quite a few entries.

Reply:
You're right, it's probably the best way to handle my own setup globally.